src/exportContracts.ts

```diff
@@ -0,0 +1,6 @@
+export {
+  Core__factory as Core,
```

Comment:
@Kastet @zatlite Did we manage to figure out why the typedoc still generates these as the former names?

Reply:
I can't see former names? https://docs.x.immutable.com/sdk-references/core-sdk-ts/1.0.0-beta.2/modules/index.contracts

Reply:
oh nice! looks good to me. @Kastet looks resolved now?